### PR TITLE
Fix getPeriodNames for globalize (T1162346)

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: js/localization/globalize/date.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ if(Globalize && Globalize.formatDate) {
             return Globalize.locale().main('dates/calendars/gregorian/' + path);
         },
 
-        getPeriodNames: function() {
-            const json = Globalize.locale().main('dates/calendars/gregorian/dayPeriods/stand-alone/wide');
+        getPeriodNames: function(format) {
+            const json = Globalize.locale().main(`dates/calendars/gregorian/dayPeriods/stand-alone/${format || 'wide'}`);
```
Comment (ksercs) [inline js/localization/globalize/date.js]:
```suggestion
            const json = Globalize.locale().main(`dates/calendars/gregorian/dayPeriods/stand-alone/${format}`);
```

File: js/localization/globalize/date.js [hunk 2]
```diff
@@ -114,8 +114,8 @@ if(Globalize && Globalize.formatDate) {
             return Globalize.locale().main('dates/calendars/gregorian/' + path);
         },
 
-        getPeriodNames: function() {
-            const json = Globalize.locale().main('dates/calendars/gregorian/dayPeriods/stand-alone/wide');
+        getPeriodNames: function(format) {
```
Comment (ksercs) [inline js/localization/globalize/date.js]:
```suggestion
        getPeriodNames: function(format = 'wide') {
```

File: js/localization/globalize/date.js [hunk 3]
```diff
@@ -114,8 +114,8 @@ if(Globalize && Globalize.formatDate) {
             return Globalize.locale().main('dates/calendars/gregorian/' + path);
         },
 
-        getPeriodNames: function() {
```
Comment (ksercs) [inline js/localization/globalize/date.js:117]: what about tests?

Comment (ksercs) [inline js/localization/globalize/date.js:117]: please, add tests for datebox localization. U can find it in this PR: https://github.com/DevExpress/DevExtreme/pull/24451

File: js/localization/globalize/date.js (hunk 1 above)
Comment (ksercs) [inline js/localization/globalize/date.js]: the issue is still reproduced when `aaaa` is used in format

Comment (GoodDayForSurf) [reply js/localization/globalize/date.js]: fixed

File: js/localization/globalize/date.js (hunk 3 above)
Comment (GoodDayForSurf) [reply js/localization/globalize/date.js:117]: added

File: js/localization/globalize/date.js (hunk 2 above)
Comment (GoodDayForSurf) [reply js/localization/globalize/date.js]: maybe it's better to have insurance from getPeriodNames(null) calling?

File: js/localization/globalize/date.js (hunk 3 above)
Comment (GoodDayForSurf) [reply js/localization/globalize/date.js:117]: added

**Review by ksercs — DISMISSED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**